mix.exs

```diff
@@ -27,9 +28,8 @@ defmodule Nectar.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
+      {:espec, "~> 1.5", only: :test},
```

Comment:
Why espec over ExUnit? ExUnit seems overwhelming more widely used and would be more familiar to potential contributors.